### PR TITLE
Adds a cowboy bundle to the nuclear ops uplink

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Tin

The bundle includes revolver, holster, armored cowboy outfit, horse, apples to tame said horse, and a complimentary lighter.

It costs 18tc, so it's a little more expensive than getting the revolver & holster on their own, but you also get a horse so it's worth it.
## Why It's Good For The Game
This was intended as a lone-ops bundle so we can finally have some lone rangers but I think having a crew of cowboy outlaws doing a "train heist" for a nuke is really funny.
## Changelog
:cl: Wallem
add: Nuclear Operatives now have ready access to ancient cowboy technology in the form of the Outlaw Bundle. Now you too can roll into town on your horse.
/:cl:

### CODE REVIEW

File: code/game/machinery/syndicatebeacon.dm [hunk 1]
```diff
@@ -146,3 +146,7 @@
 /obj/item/sbeacondrop/clownbomb
 	desc = "A label on it reads: <i>Warning: Activating this device will send a silly explosive to your location</i>."
 	droptype = /obj/machinery/syndicatebomb/badmin/clown
+
+/obj/item/sbeacondrop/horse
+	desc = "A label on it reads: <i>Warning: Activating this device will send a live horse to your location. Remember that horses cannot survive in space!</i>"
```
Comment (carlarctg) [inline code/game/machinery/syndicatebeacon.dm:151]: update

File: code/game/objects/items/storage/uplink_kits.dm [hunk 2]
```diff
@@ -789,6 +789,21 @@
...
+/obj/item/storage/box/syndie_kit/cowboy
+	name = "western outlaw pack"
+	desc = "Contains everything you'll need to be the rootin' tootin' cowboy you always wanted. Either play the Lone Ranger or go in with your posse of outlaws."
+
+/obj/item/storage/box/syndie_kit/cowboy/PopulateContents()
+	generate_items_inside(list(
+		/obj/item/clothing/shoes/cowboy/black/syndicate= 1,
+		/obj/item/clothing/head/cowboy/black/syndicate = 1,
+		/obj/item/storage/belt/holster/nukie/cowboy/full = 1,
+		/obj/item/clothing/under/costume/dutch/syndicate = 1,
```
Comment (carlarctg) [inline code/game/objects/items/storage/uplink_kits.dm:801]: add a hat stabilizer module

File: code/modules/clothing/shoes/cowboy.dm [hunk 3]
```diff
@@ -6,6 +6,10 @@
 	custom_price = PAYCHECK_CREW
 	var/max_occupants = 4
 	can_be_tied = FALSE
+	/// Do these boots have spur sounds?
+	var/has_spurs = FALSE
+	/// The jingle jangle jingle of our spurs
+	var/list/spur_sound = list('sound/effects/footstep/spurs1.ogg'=1,'sound/effects/footstep/spurs2.ogg'=1,'sound/effects/footstep/spurs3.ogg'=1)
```
Comment (carlarctg) [inline code/modules/clothing/shoes/cowboy.dm:12]: document this

File: code/modules/mob/living/basic/farm_animals/pony.dm [hunk 4]
```diff
@@ -86,3 +96,18 @@
 		/datum/ai_planning_subtree/basic_melee_attack_subtree,
 		/datum/ai_planning_subtree/random_speech/pony
 	)
+
+// A stronger horse is required for our strongest cowboys.
+/mob/living/basic/pony/syndicate
```
Comment (carlarctg) [inline code/modules/mob/living/basic/farm_animals/pony.dm:101]: give it the spacewalk trait?

**Comment by carlarctg** [comment]
please add a lazarus to the kit

File: code/modules/clothing/shoes/cowboy.dm (hunk 3 above)
Comment (Wallemations) [reply code/modules/clothing/shoes/cowboy.dm:12]: It is documented???

**Comment by F-e-r-n** [comment]
Love the idea but do we need a horse spacesuit for it?

File: code/modules/clothing/shoes/cowboy.dm (hunk 3 above)
Comment (carlarctg) [reply code/modules/clothing/shoes/cowboy.dm:12]: it is not in the PR body, you need to announce all the features you added in code (in this case spur jingles)

**Comment by Wallemations** [comment]
> Love the idea but do we need a horse spacesuit for it?

no

**Comment by Loafin34** [comment]
are the horses space-proof?

**Comment by Wallemations** [comment]
> are the horses space-proof?

Syndicate ones are, though they can't run through space

**Comment by Loafin34** [comment]
also, please please add "horse glue" to make it so that dastardly unbucklers can't steal a prized space-horse

**Comment by Wallemations** [comment]
> also, please please add "horse glue" to make it so that dastardly unbucklers can't steal a prized space-horse


I can just override their tamable component to make them single-person tamed
Iirc that'll make it so only one person can ride them

**Comment by Livrah** [comment]
Love more themed bundles.
> I can just override their tamable component to make them single-person tamed 
Every tamed mobs should have it.

File: code/modules/clothing/shoes/cowboy.dm (hunk 3 above)
Comment (Wallemations) [reply code/modules/clothing/shoes/cowboy.dm:12]: it is, I said you get a cowboy outfit

**Pull request merged**